crates/gpui/src/platform/linux/wayland/client.rs

```diff
@@ -476,7 +476,7 @@ impl WaylandClient {
             .as_ref()
             .map(|primary_selection_manager| primary_selection_manager.get_device(&seat, &qh, ()));
 
-        let mut cursor = Cursor::new(&conn, &globals, 24);
+        let mut cursor = Cursor::new(&conn, &globals, 24, 2);
```

Comment:
It seems like this number should be dynamically determined by the compositor rather than being hard-coded. A FIXME comment would make it clear that there's still work to do to handle other situations.